src/lib/Libutil/pbs_aes_encrypt.c

```diff
@@ -33,27 +33,28 @@ extern unsigned char pbs_aes_iv[];
 int
 pbs_encrypt_data(char *uncrypted, int *credtype, size_t len, char **crypted, size_t *outlen)
 {
-        EVP_CIPHER_CTX ctx;
+        EVP_CIPHER_CTX *ctx = EVP_CIPHER_CTX_new();
```

Comment:
Hey @arungrover,
First thanks a lot reviewing this. 😄 
This change is due to there is change in latest version of openssl about how we can access internal structure of API.
Please have a look at [here](https://github.com/openssl/openssl/issues/962#issuecomment-208792020) for more information.
Please let me know if you need more information.